packages/core/src/policy/types.ts

```diff
@@ -117,6 +117,12 @@ export interface PolicyRule {
    * Default is 0.
    */
   priority?: number;
```

Comment:
not a new issue but this should not be nulllable either and the default should be applied here. we can address that in a followup for simplicity.